### DISCUSSION TITLE
[Config Support]: Still can't open Frigate after removing a :ro in docker-compose.yaml

### DISCUSSION THREAD

**Discussion by tomhusband** [blakeblackshear/frigate#11482] [Config Support]
### Describe the problem you are having

I've been having a problem opening Frigate. I read [here](https://github.com/blakeblackshear/frigate/issues/9539) and [here](https://github.com/blakeblackshear/frigate/issues/9529) about similar problems when the `- /opt/frigate/config:/config: ro`file is read only. The solution is to remove the `: ro`. I did that, rebooted everything but, still, nothing opens when I go to http://192.168.68.100:5000/. I really thought I had the fix. 

I used this [tutorial](https://www.homeautomationguy.io/blog/running-frigate-on-proxmox) to install.

The log the log from Portainer shows `[Errno 30] Read-only file system: '/config/model_cache'`. Would that be the problem? I'm not sure where that file is.

Any suggestions on where to look now? I obviously missed something.

### Version

Latest version I assume.I can't open the app

### Frigate config file

```yaml
mqtt:
  enabled: false
ffmpeg:
  hwaccel_args: preset-vaapi
detectors:
  coral:
    type: edgetpu
    device: usb
#Global Object Settings
objects:
  track:
    - person
  filters:
    person:
      min_area: 5000
      max_area: 100000
cameras:
  driveway_cam:
    ffmpeg:
      inputs:
        - path: rtsp://admin:xxxxx.xxxxx@192.168.68.60:554/cam/realmonitor?channel=1&su>
          roles:
            - detect
            - record
            - rtmp
    detect:
      width: 1280
      height: 720
      fps: 10

    objects:
      track:
        - car
            mask
              - 1164,611,610,720,493,211,838,71
        - person
        - dog
        - cat
        - bicycle
      filters:
        person:
          min_score: 0.6
          threshold: 0.7
        dog:
          min_score: 0.6
          threshold: 0.75
    record:
      enabled: True
      retain:
      enabled: True
      retain:
        days: 0
        mode: active_objects
      events:
        retain:
          default: 3
          mode: active_objects

    motion:
      improve_contrast: True
```

### Relevant log output

```shell
Log from Portainer after I removed the : ro

2024-05-22 13:47:57.616585995  13:47:57.616 INF [api] listen addr=:1984
2024-05-22 13:47:58.027706356  [Errno 30] Read-only file system: '/config/model_cache'
2024-05-22 13:47:58.149758239  [INFO] Service Frigate exited with code 1 (by signal 0)
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service nginx: stopping
s6-rc: info: service go2rtc-healthcheck: stopping
2024-05-22 13:47:58.158505291  [INFO] The go2rtc-healthcheck service exited with code 256 (by signal 15)
s6-rc: info: service go2rtc-healthcheck successfully stopped
2024-05-22 13:47:58.184495687  [INFO] Service NGINX exited with code 0 (by signal 0)
s6-rc: info: service nginx successfully stopped
s6-rc: info: service nginx-log: stopping
s6-rc: info: service frigate: stopping
s6-rc: info: service frigate successfully stopped
s6-rc: info: service go2rtc: stopping
s6-rc: info: service frigate-log: stopping
2024-05-22 13:47:58.186541097  exit with signal: terminated
s6-rc: info: service nginx-log successfully stopped
s6-rc: info: service frigate-log successfully stopped
2024-05-22 13:47:58.189882416  [INFO] The go2rtc service exited with code 0 (by signal 0)
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

USB

### Any other information that may be helpful

Current docker-compose.yaml file:

version: '3.9'

services:

  frigate:
    container_name: frigate
    privileged: true
    restart: unless-stopped
    image: ghcr.io/blakeblackshear/frigate:stable
    shm_size: "128mb" # update for your cameras based on calculation above
    devices:
      - /dev/bus/usb:/dev/bus/usb
      - /dev/dri/renderD128 # for intel hwaccel, needs to be updated for your hardware
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /opt/frigate/config:/config
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "5000:5000"
      - "1935:1935" # RTMP feeds
    environment:
      FRIGATE_RTSP_PASSWORD: "xxxxx.xxxxx"

**Comment by NickM-27** [comment]
yes that is a problem, in general it means that /config is read only, something to do with teh host file system being read only (`/opt/frigate/config`)

**Reply by tomhusband** [reply]
The host file system? Meaning Proxmox? How can I change that?

**Reply by NickM-27** [reply]
Yes, something in proxmox. I don't know, I don't use proxmox.

**Reply by tomhusband** [reply]
I guess I don't understand why it would be like that. I certainly didn't specify that. There must lots of folks who have used this tutorial to install Frigate in Docker in a Proxmox LXC. 

What do you use for Frigate?

**Reply by NickM-27** [reply]
yes, and this issue (after fixing the :ro) has only been seen happening to proxmox users, so it certainly seems to be related to that.

I installed it in docker which runs directly on linux host, like the docs recommend for easy installation

**Reply by tomhusband** [reply]
Something must have changed because I used the Docker/Proxmox container method in a previous install and it ran beautifully.
Pretty frustrating. Guess I'll look for another way.

Thanks

**Reply by partytimeexcellent** [reply]
@tomhusband Did you ever sort this out? just ran into the same issue after following the same tutorial and changing /opt/frigate/config:/config:ro to :rw
